File: Python/Colecciones/ejercicio10/Ejercicio10.py
diccionario = {}

diccionario['rojo'] = 'rojo'
diccionario['azul'] = 'blue'
diccionario['verde'] = 'green'
diccionario['negro'] = 'black'
diccionario['blanco'] = 'white'
diccionario['barco'] = 'boat'
diccionario['coche'] = 'car'
diccionario['autobus'] = 'bus'
diccionario['bicicleta'] = 'bike'
diccionario['moto'] = 'motorcycle'
diccionario['perro'] = 'dog'
diccionario['gato'] = 'cat'
diccionario['caballo'] = 'horse'
diccionario['mono'] = 'monkey'
diccionario['pajaro'] = 'bird'
diccionario['cama'] = 'bed'
diccionario['ventana'] = 'windows'
diccionario['silla'] = 'chair'
diccionario['mesa'] = 'desk'
diccionario['ordenador'] = 'computer'

palabra = input("Plabra en español: ")

if diccionario.__contains__(palabra):
    print("Palabra en ingles: " , diccionario.get(palabra))

else :
    print("Esa palabra no se encuentra en el diccionario")

File: Java/Diccionarios y colecciones/ejercicio10/Ejercicio10.java
package ejercicio10;

import java.util.HashMap;
import java.util.Scanner;

/**
 * 10. Crea un mini-diccionario español-inglés que contenga, al menos, 20 palabras
 * (con su correspondiente traducción). Utiliza un objeto de la clase HashMap
 * para almacenar las parejas de palabras. El programa pedirá una palabra en
 * español y dará la correspondiente traducción en inglés.
 * 
 * @author Juan Manuel Gonzalez Profumo
 *
 */
public class Ejercicio10 {
	public static void main(String[] args) {
		Scanner sc = new Scanner(System.in);
		HashMap<String, String> diccionario = new HashMap<String, String>();
		
		diccionario.put("rojo", "red");
		diccionario.put("azul", "blue");
		diccionario.put("verde", "green");
		diccionario.put("negro", "black");
		diccionario.put("blanco", "white");
		diccionario.put("barco", "boat");
		diccionario.put("coche", "car");
		diccionario.put("autobus", "bus");
		diccionario.put("bicicleta", "bike");
		diccionario.put("moto", "motorcycle");
		diccionario.put("perro", "dog");
		diccionario.put("gato", "cat");
		diccionario.put("caballo", "horse");
		diccionario.put("mono", "monkey");
		diccionario.put("pajaro", "bird");
		diccionario.put("cama", "bed");
		diccionario.put("ventana", "windows");
		diccionario.put("silla", "chair");
		diccionario.put("mesa", "desk");
		diccionario.put("ordenador", "computer");
		
		System.out.println("Palabra en español: ");
		String palabra = sc.nextLine();
		
		if (diccionario.containsKey(palabra)) {
			System.out.println("Palabra en ingles: " + diccionario.get(palabra));
		}
		
		else {
			System.out.println("Esa palabra no se encuentra en el diccionario");
		}
	}

}
